src/test/scala/temple/generate/target/openapi/OpenAPIGeneratorTest.scala

```diff
@@ -138,11 +140,39 @@ class OpenAPIGeneratorTest extends FlatSpec with Matchers {
         |        format: int32
         |      required: true
         |      description: ID of the match to perform operations on
+        |    delete:
+        |      summary: Delete a single match
+        |      tags:
+        |      - Match
+        |      responses:
+        |        '500':
+        |          $ref: '#/components/responses/Error500'
+        |        '404':
+        |          $ref: '#/components/responses/Error404'
+        |        '401':
+        |          $ref: '#/components/responses/Error401'
+        |        '400':
+        |          $ref: '#/components/responses/Error400'
+        |        '200':
```

Comment:
Previous responses are in increasing order (200, 400, 401, 500), whereas these are in decreasing. Should we be consistent?

Reply:
They’re in decreasing order purely as a side-effect of the use of unordered maps. Sorting that out now